app/BUILD.gn

```diff
@@ -26,9 +26,12 @@ grit("brave_generated_resources_grit") {
     "root_gen_dir=" + rebase_path(root_gen_dir, root_build_dir),
   ]
 
-  deps = [
-    "//brave/components/services:brave_content_manifest_overlays",
-  ]
+  if (!is_android) {
+    deps = [
+      "//brave/components/services:brave_content_manifest_overlays",
```

Comment:
can you please give me more context about this change?

Reply:
It wasn't compilable with this dependency on Android. Since it's not used on Android it was excluded.

Reply:
looks like not related to rewards

Reply:
Reverted